packages/nc-gui/components/account/Token.vue

```diff
@@ -75,17 +90,19 @@ const deleteToken = async (token: string): Promise<void> => {
 }
 
 const generateToken = async () => {
+  isTokenNameEmpty.value = !selectedTokenData.value.description?.length || selectedTokenData.value.description?.length > 255
```

Comment:
if `selectedTokenData.value.description?.length > 255` is true, my token name is not empty but the variable it means it is.

Henec, need a better name.